README.md

```diff
@@ -32,27 +32,31 @@ achieved through [Gremlin](https://tinkerpop.apache.org/gremlin.html)(a powerful
 
 ## Quick Start
 
-### 1. Docker Way (Convenient for Test)
```

Comment:
Thanks, but currently we need to keep the `Test` flag for ASF rule

--> Docker is not the best choice recommended by ASF, so we need to make a statement to remind users

Reply:
Got it, thanks.  Added `For Test`